modules/ibi-create-standalone-config-iso.adoc

```diff
@@ -0,0 +1,229 @@
+// Module included in the following assemblies:
+//
+// * edge_computing/ibi-edge-image-based-install.adoc 
+
+:_mod-docs-content-type: PROCEDURE
+[id="create-standalone-config-iso_{context}"]
+= Deploying a {sno} cluster using the {product-title} installation program
+
+You can use the {product-title} installation program to configure and deploy a host that you preinstalled with an image-based installation. To configure the target host with site-specific details, you must create the following resources:
+
+* The `install-config.yaml` installation manifest.
+* The `image-based-config.yaml` manifest.
+
+The `openshift-install` installation program consumes these resources to generate a configuration ISO that you can attach to your preinstalled target host and complete the deployment.
+
+[NOTE]
+====
+For more information about the specifications for the `image-based-config.yaml` manifest, see the section _Reference specifications for the image-based-config.yaml manifest_.
+====
+
+.Prerequisites
+
+* You preinstalled a host with {sno} using an image-based installation.
+* You downloaded the latest version of the `openshift-install` installation program.
+* You created a pull secret to authenticate pull requests. See the _Additional resources_ section for more information about using image pull secrets.
+
+.Procedure
+
+. Create a working directory by running the following:
++
+[source,terminal]
+----
+$ mkdir ibi-config-iso-workdir <1>
+----
+<1> Replace `ibi-config-iso-workdir` with the name of your working directory.
+
+. Create the installation manifest:
+
+.. Create a YAML file that defines the `install-config` manifest:
++
+.Example `install-config.yaml` file
+[source,yaml]
+----
+apiVersion: v1
+metadata:
+  name: sno-cluster-name
+baseDomain: host.example.com
+compute:
+  - architecture: amd64
+    hyperthreading: Enabled
+    name: worker
+    replicas: 0
+controlPlane:
+  architecture: amd64
+  hyperthreading: Enabled
+  name: master
+  replicas: 1
+networking:
+  clusterNetwork:
+  - cidr: 10.128.0.0/14
+    hostPrefix: 23
+  machineNetwork:
+  - cidr: 10.6.85.0/24
+  networkType: OVNKubernetes
+  serviceNetwork:
+  - 172.30.0.0/16
+platform:
+  none: {}
+fips: false
+cpuPartitioningMode: "AllNodes"
+pullSecret: '{"auths":{"... REDACTED ..."}}}'
+sshKey: 'ssh-rsa AAAAB3…REDACTED…'
+----
+
+.. Save the file in your working directory. 
+
+. Optional. Create a configuration template in your working directory by running the following command:
++
+[source,terminal]
+----
+$ openshift-install image-based create config-template --dir ibi-config-iso-workdir/
+----
++
+.Example output
+[source,terminal]
+----
+INFO Config-Template created in: ibi-config-iso-workdir
+----
++
+The command creates the `image-based-config.yaml` configuration template in your working directory:
```

Comment:
Again seems pointless to include this example output since we're giving a full example in the next step